tests/Sulu/Component/Content/Mapper/ContentMapperTest.php

```diff
@@ -3187,6 +3187,122 @@ public function testContentTypeSwitch()
         $this->mapper->save($testSiteData, 'internal-link', 'sulu_io', 'en', 1, true, $uuid);
     }
 
+    /**
+     * It should delete a node which has children with history.
+     * It should not throw an exception
```

Comment:
Well, it should be `@unexpectedException` which does not exist :package: 

But the idea here is to *encourage* specification driven tests. I have taken this from PHPSpec and have done it in all my projects for 2 months, and it has made a big difference to me.

i.e. the first thing you do is write a sentance which describes what should happen.

phpspec would do it as:

````
public function is_should_delete_a_node_which_as_children_with_history()
````

But I find that to constrictive.